BrokerApiNode/test/http.apiKey.test.js

```diff
@@ -0,0 +1,28 @@
+const HttpApiPublish = require('../src/http-api-publish')
+const BrokerConfig = require('../src/config-api-key.json')
+const { expectConsoleLogs, startBroker, wait} = require('./util')
+
+describe('HTTP:Api-Key:Publish', () => {
```

Comment:
I loaded the v31 broker and these tests seem to fail, I wasn't able to test on an earlier version.  

● HTTP:Api-Key:Publish › should exercise the `publish` example
    must have required property 'client'`